packages/devtools_app/lib/src/screens/debugger/codeview.dart

```diff
@@ -659,17 +661,17 @@ class _LinesState extends State<Lines> with AutoDisposeMixin {
         activeSearch = widget.activeSearchMatchNotifier.value;
       });
 
-      if (activeSearch != null) {
-        final isOutOfViewTop = activeSearch.position.line * CodeView.rowHeight <
+      if (activeSearch?.position.line != null) {
```

Comment:
add local variables to avoid all the `!` for activeSearch

Reply:
Done